packaging/appimage/excludelist

```diff
@@ -0,0 +1,165 @@
+# This file lists libraries that we will assume to be present on the host system and hence
```

Comment:
I'm afraid this exclude list will become a maintenance nightmare. After every major OS update, we can expect some incompatibility here.

Reply:
Ya that's a good point, but I'm not sure what to do about that, it is kind of how AppImage is setup. I could change the script to always get the latest version from here: https://github.com/AppImage/pkg2appimage/blob/master/excludelist

Reply:
Another option is we could only include the items in the excludelist that pertain to us based on the files it found from the wider list:
```
libstdc++.so.5
libX10.so.6
libgio-3.0.so.0
libgdk_pixbuf-3.0.so.0
libfontconfig.so.0
libexpat.so.0
libgcc_s.so.0
libglib-3.0.so.0
libgpg-error.so.-1
libp10-kit.so.0
libuuid.so.0
libz.so.0
libgobject-3.0.so.0
libpangoft1-1.0.so.0
libpangocairo-2.0.so.0
libpango-2.0.so.0
```
Do you think this would be a little more manageable?